plugins/proxy-port-forward.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: proxy-port-forward
+spec:
+  version: v1.0.0
+  homepage: https://github.com/kvaps/kubectl-proxy-port-forward
+  shortDescription: Proxy to remote host via Kubernetes
+  description: |
+    Start socat with proxy to remote host in Kubernetes and forward one or more local ports to it.
```

Comment:
Q: Are there no `caveats` to this tool at all, like maybe the pod requires `socat`? 

In the code I see things like `kubectl delete pod`, which I would not expect this plugin to do. I think this field is a good opportunity to make it clear to users how it works even more.

Reply:
The approach is similar to `node-shell` plugin.

A plugin spawns a new pod with socat to proxy all requests to specified remote hosts, which are available only from Kubernetes cluster.
After usage the pod obviously should be removed.

Reply:
I think you explained it really well here. This can be made clear in the description too.